docs/DeveloperGuide.adoc

```diff
@@ -721,17 +721,50 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+|`* * *` | user | be warned before execution of a hazardous command | prevent from disastrous mistakes
+|`* * *` | user | change the theme of the GUI (dark/bright) | prevent eye strain or simply customize
+|`* * *` | user | validate persons details when adding |  prevent any mistakes
+|`* * *` | user | input more than 1 phone number | can store home + office + mobile numbers easily
+|`* * *` | user | search for people via different fields (address, phone number etc.) | easily view people of same area of residence
+|`* * *` | user | allow mass deletion | easy cleanup
+|`* * *` | user | have a favourites tag that is always easily accessible | easily access most frequent contacts
+|`* * *` | forgetful user | show my phone number | so that I can give it to my friend when they ask
+|`* *` | user with many persons in the address book | apply filter: category, tag, name | easily view and process only certain group
+|`* *` | user | redo the command | correct the mistake of undoing too many commands
+|`* *` | user | view the history of commands | check what I have done in the past
+|`* *` | user | import persons | share with other users and migrate from other systems
+|`* *` | user | export persons | share with other users and migrate from other systems
+|`* *` | user | use password to get access to my contacts | protect my data
+|`* *` | user | clear the address book | start anew
+|`* *` | user | see suggestions while typing | type commands faster
+|`* *` | impatient user | get the previously entered commands into the input field | type commands faster
+|`* *` | new user | use a tutorial | learn the features of the app quickly
+|`* *` | user | remove/categorize/tag all the persons selected by a filter | process multiple contacts quickly
+|`* *` | impatient user | use hotkeys | navigate faster
+|`* *` | user | add new fields to all the persons | customize the App
+|`* *` | user | sort by frequency of access | easily access most frequent contacts
+|`* *` | forgetful user | have a reminder which reminds me to call someone at certain times | call my contacts if needed
+|`* *` | busy user | save appointment with people in my addressbook and send them a message about the appointment | set up appointment with my contacts easily
+|`* *` | stealthy users | have hidden contacts which can only be shown upon typing a secret phrase | hide my other girlfriends from my girlfriend
+|`* *` | stealthy users | delete all hidden contacts after the passphrase is entered wrongly 5 times | noone can excess my hidden contacts
+|`* *` | user | send my contact to people in my address book | so that I can give it to my friend when they ask
+|`* *` | user | type names in either case | it can auto-capitalise the first letter of all names
+|`*` | user with many persons in the address book | sort persons by name | locate a person easily
+|`*` | clumsy user | undo a command | correct a mistake that was previously executed
```

Comment:
Try not to use adjectives with negative connotation to describe a user. You can always say sth like "to avoid ...".